ui/inspectors/system-section.reel/docker-settings.reel/docker-settings.js

```diff
@@ -9,16 +9,28 @@ var Component = require("montage/ui/component").Component,
  * @extends Component
  */
 exports.DockerSettings = Component.specialize(/** @lends DockerSettings# */ {
+
+    apiForwardingAddress: {
+        value: null
+    },
+
     enterDocument: {
         value: function(isFirstTime) {
             var self = this;
+                this.apiForwardingAddress = [];
             if(isFirstTime) {
                 this.isLoading = true;
                 this.application.dockerSettingsService.getDockerHostQueryData().then(function(dockerHosts) {
                     self.dockerHosts = dockerHosts;
                 })
                 this.application.dockerSettingsService.getDockerConfigData().then(function (dockerConfig) {
-                    self.object = dockerConfig
+                    self.object = dockerConfig;
+                    var api_forwarding_list = dockerConfig.api_forwarding.split(",")
+                    if(dockerConfig.api_forwarding && dockerConfig.api_forwarding.length > 0) {
+                        for (var i in api_forwarding_list) {
```

Comment:
As of today, `for ... in` constructs in Javascript are known to have bad performances, please use a classical `for` loop instead